SobokSobok/SobokSobok/Presentation/Medicine/AddMedicineSheet/AddMedicineSheet.swift

```diff
@@ -0,0 +1,68 @@
+//
+//  AddMedicineSheet.swift
+//  SobokSobok
+//
+//  Created by 김승찬 on 2022/01/09.
+//
+
+import UIKit
+import EasyKit
+
+class AddMedicineSheet: UIViewController {
```

Comment:
```suggestion
final class AddMedicineSheet: UIViewController {
```

이 친구도 상속은 안 받을 것 같기 때문에 final 키워드를 붙여주세요!

Reply:
네넹~